docs/gettingstarted/structuring.md

```diff
@@ -25,6 +25,69 @@ What is `@Mod`?
 
 This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
 
+All components of @Mod() are written inside the parantheses and chained together. There's only ever a single @Mod in your entire mod, sitting on top of your entry class.
```

Comment:
> About multiple mods: I can think of a few, but all I am saying is that you should not say "there is only ever one @Mod". I would put it like this: "There is one @Mod per mod.".

That seems like a reasonable adjustment. It's useful in readability and hints at the idea that there might be more to it.

In general I'm trying to keep this information relevant to the task at hand, which includes minor dips into related topics. Whether or not that counts as doc or tut remains to be decided.